### ISSUE TITLE
Regarding Co Segmentation

### ISSUE THREAD

**Issue by saisusmitha** [suyukun666/UFO#9]
Hi. Which part of the code exactly deals with the co-segmentation part? Is there a way where I can only do this co-segmentation?

**Comment by Gi-gigi** [comment]
Dear Author, I am unsure about the training dataset COCO-SEG related to co-segmentation and co-object detection. Could you please provide a download link for the dataset if it's available? @suyukun666

**Comment by saisusmitha** [comment]
@suyukun666 Yes can you please share how to download the training data for cosegmentation
